packages/cli/src/lab.ts

```diff
@@ -0,0 +1,86 @@
+import { createReadStream, createWriteStream, type PathLike } from 'node:fs';
+import { readdir } from 'node:fs/promises';
+import { fileURLToPath } from 'node:url';
+import mime from 'mime';
+import { serve } from "@hono/node-server"
```

Comment:
its not necessary to keep hono as a dependency. i find it easier to develop with the whatwg request and response object. but now that its built, it can be easily translated into the node apis.

Reply:
Oh I love hono. Keep it